exchanges/localbitcoins/localbitcoins_wrapper.go

```diff
@@ -286,8 +288,30 @@ func (l *LocalBitcoins) GetFundingHistory() ([]exchange.FundHistory, error) {
 }
 
 // GetExchangeHistory returns historic trade data since exchange opening.
-func (l *LocalBitcoins) GetExchangeHistory(p currency.Pair, assetType asset.Item) ([]exchange.TradeHistory, error) {
-	return nil, common.ErrNotYetImplemented
+func (l *LocalBitcoins) GetExchangeHistory(req *trade.HistoryRequest) ([]trade.History, error) {
+	v := url.Values{}
+	if req.TradeID != "" {
+		v.Set("since", req.TradeID)
+	}
+
+	fPair := l.FormatExchangeCurrency(req.Pair, req.Asset)
+	t, err := l.GetTrades(fPair.String(), v)
```

Comment:
![image](https://user-images.githubusercontent.com/9261323/81357476-14f38380-9117-11ea-8a28-426ce7d2a2c4.png)

`GetTrades` is also not tested